manage.py

```diff
@@ -186,6 +199,19 @@ def create_user(email, role):
     db.session.add(user)
     db.session.commit()
 
+def __parse_tech_names__(tech_str):
```

Comment:
Regarding the double underscore naming convention:

- http://stackoverflow.com/questions/1301346/the-meaning-of-a-single-and-a-double-underscore-before-an-object-name-in-python

I've named a few things like this throughout the project, but now believe it's the wrong practice.  I believe `__name__` is typically reserved for python itself so that it does not conflict with user names.

This should probably be named `def _parse_tech_names(tech_str):` with a single leading underscore and no trailing underscores.

A couple other places I've done this:
- `security_monkey.views.__check_auth__`
- `security_monkey.auditors.security_group.__port_for_rule__`